docs/docs/documentation/getting-started/quick-start.md

```diff
@@ -88,6 +93,10 @@ You can use different gateway interfaces to communicate with the system such RES
 
 This will provide a simple chat interface where you can interact with the Agent Mesh. Try some commands like `What is the capital of France?` or `Give me a diagram of the oAuth process`.
 
+:::tip
```

Comment:
Same comment here, I would prefer to remove this tip for now since we already mention 127.0.0.1:5001 above. We can add it back if we get feedback that multiple people are tripping over this. This might go better in a "troubleshooting section".